scapy/layers/inet6.py

```diff
@@ -1737,18 +1737,41 @@ class _ICMPv6NDGuessPayload:
 
     def guess_payload_class(self, p):
         if len(p) > 1:
-            return icmp6ndoptscls.get(orb(p[0]), Raw)  # s/Raw/ICMPv6NDOptUnknown/g ?  # noqa: E501
+            return icmp6ndoptscls.get(orb(p[0]), ICMPv6NDOptUnknown)
```

Comment:
I think it would maybe be useful if the known options that can't be parsed properly turned into `ICMPv6NDOptUnknown` or something like that by analogy with `tshark` showing options and their bytes. One recent example where it could have helped would be https://lists.debian.org/debian-security-announce/2024/msg00123.html (which was (partly) powered by `scapy` FWIW). I'll try to figure out if it makes sense.